### ISSUE TITLE
Zip mojo problem with assembly plugin

### ISSUE THREAD

**Issue by jeffmaury** [asciidoctor/asciidoctor-maven-plugin#145]
Hello,

I am facing the following issue: I'm building a ZIP file of HTML produced by the Zip mojo and it's working file. But this artifact is then used and unzipped by the assembly plugin. Then the content of the assembly is processed by another Maven plugin (IZpack). The problem I have is that the files extracted from the ZIP file have no read/write/execute flags set. I have looked at the ZIP file and the Host OS is set to FAT and it seems to be no attributes.
I have this problem only on Linux not on Windows
![image](https://cloud.githubusercontent.com/assets/695993/5739875/657f4320-9bfe-11e4-94e5-8ca407a74395.png)

**Comment by mojavelinux** [comment]
I understand the problem you are having but I don't see how it has anything
to do with the Asciidoctor plugin. Are you suggesting a correlation, or
have you filed the issue with the wrong project?

**Comment by jeffmaury** [comment]
No, it is correlated as the ZIP file is produced by the asciidoctor zip mojo.

**Comment by LightGuard** [comment]
This is exactly why I didn't want to have one plugin do the work of many.

On Wed, Jan 14, 2015 at 12:48 PM, Jeff MAURY <notifications@github.com>
wrote:

> No, it is correlated as the ZIP file is produced by the asciidoctor zip
> mojo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor-maven-plugin/issues/145#issuecomment-69978991>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by mojavelinux** [comment]
Hahaha. I didn't even realize the Asciidoctor plugin produced a zip. Shouldn't that be handled by the assembly plugin (as I think Jason is suggesting)?

**Comment by LightGuard** [comment]
That was my take originally.

On Wed, Jan 14, 2015 at 1:28 PM, Dan Allen <notifications@github.com> wrote:

> Hahaha. I didn't even realize the Asciidoctor plugin produced a zip.
> Shouldn't that be handled by the assembly plugin (as I think Jason is
> suggesting)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor-maven-plugin/issues/145#issuecomment-69985655>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by jeffmaury** [comment]
That was how I did a workaround for the problem (asciidoctor generating the HTML files and the assembly plugin buiding the Zip). But I wanted to signal the problem and get feedback to/from the community

**Comment by LightGuard** [comment]
I haven't heard of anyone else (besides the person who put it in) who was
using it. I'm not even sure if it's documented (probably not as you hit
this issue). I'm tempted to say let's deprecate it.

On Wed, Jan 14, 2015 at 2:40 PM, Jeff MAURY <notifications@github.com>
wrote:

> That was how I did a workaround for the problem (asciidoctor generating
> the HTML files and the assembly plugin buiding the Zip). But I wanted to
> signal the problem and get feedback to/from the community
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor-maven-plugin/issues/145#issuecomment-69997463>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by jeffmaury** [comment]
+1 for deprecation

**Comment by abelsromero** [comment]
I think the same, we should stick to what we do best. That's making awsome documents :+1: 
Currently we could set the Mojo as "unsuported" and remove it in some future versions.

**Comment by LightGuard** [comment]
There are some other gems in there that I think we should do the same. Like
the server that's in there :)

On Wed, Jan 14, 2015 at 2:54 PM, Abel Salgado Romero <
notifications@github.com> wrote:

> I think the same, we should stick to what we do best. That's making awsome
> documents [image: :+1:]
> Currently we could set the Mojo as "unsuported" and remove it in some
> future versions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor-maven-plugin/issues/145#issuecomment-69999757>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by abelsromero** [comment]
Sorry, Zip mojo is marked for removal at some point and no longer supported.
